gnss/coord_system.py

```diff
@@ -1,19 +1,23 @@
 # Copyright (C) 2017 Swift Navigation Inc.
-# Contact: Swift Navigation <dev@swiftnav.com> 
+# Contact: Swift Navigation <dev@swiftnav.com>
 # This source is subject to the license found in the file 'LICENSE' which must
 # be be distributed together with this source. All other rights reserved.
 
 
+from typing import Any, Final, List, Tuple, Union
 import numpy as np
 
-WGS84_A = 6378137.0
-WGS84_IF = 298.257223563
-WGS84_F = (1 / WGS84_IF)
-WGS84_E = np.sqrt(2 * WGS84_F - WGS84_F * WGS84_F)
-WGS84_B = (WGS84_A * (1 - WGS84_F))
+WGS84_A = 6378137.0  # type: Final[float]
```

Comment:
Ah I hate this style of typing syntax.  Could we drop 2.7 support?

Reply:
https://pythonclock.org/

Reply:
yeah if we drop python 2.7 then I can also use numpy.typing (that package doesn't exist with python 2)

@akleeman do you have any issues with this becoming a python 3 only package?

Reply:
I have no objections, I don't think there are many people using this package and considering 2.7 is no longer maintained I think it's fair game.